src/panels/config/zone/ha-config-zone.ts

```diff
@@ -233,6 +238,9 @@ export class HaConfigZone extends SubscribeMixin(LitElement) {
 
   protected firstUpdated(changedProps: PropertyValues) {
     super.firstUpdated(changedProps);
+    this._canEditCore = ["storage", "default"].includes(
```

Comment:
And is admin

Reply:
You can't access zone config if you're not an admin, right?

Reply:
Right !